core/src/test/scala/org/broadinstitute/dsde/rawls/StatsDTestUtils.scala

```diff
@@ -59,4 +59,15 @@ trait StatsDTestUtils { this: MockitoSugar with Eventually with RawlsTestUtils =
 
   protected def expectedSubmissionStatusMetric(workspaceName: WorkspaceName, submissionStatus: SubmissionStatus, expectedTimes: Int): (String, String) =
     (s"test.workspace.${workspaceName.toString.replace('/', '.')}.submissionStatus.${submissionStatus.toString}", expectedTimes.toString)
+
+  // TODO don't repeat the code from HttpMethodExpansion.makeName() / UriExpansion.makeName()
```

Comment:
Yeah. :) Maybe `expectedHttpRequestCounterMetric` could just take the expected URI path as a `String`? Then at least we're comparing things directly.

Also I'm wondering whether this passes since above I saw `path.toString.replace('/', '_')`..